modules/op-using-secrets-and-workspaces.adoc

```diff
@@ -0,0 +1,367 @@
+// This module is included in the following assembly:
+//
+// *openshift-docs/cicd/pipelines/authenticating-pipelines-and-tasks-using-secrets.adoc
+
+[id="op-using-secrets-and-workspaces_{context}"]
+= Using secrets and workspaces
+
+Using Secrets and Workspaces in the Tekton pipeline provides a flexible approach to managing credentials in Tekton pipelines. You can bind a Secret to a Workspace, and then use that Secret content in your Task. 
+
+The following examples illustrate how to use Secrets and Workspaces.
+
+== git-clone Task
+
+This approach involves creating a task called `git-clone`, which clones a git repository using SSH authentication. The following are the steps to use Secrets and Workspace in Tekton Pipelines:
```

Comment:
We already ship `git-clone` clustertask, can't we just show how it looks like and describe how to use secrets with it?

Reply:
Hello @vdemeester , can we use the already existing `git-clone` clustertask in this case?

Reply:
@ppitonak , according to [tekton documentation](https://tekton.dev/docs/pipelines/tasks/#task-vs-clustertask), clustertasks are deprecated. As Red Hat, are we still shipping them downstream?

Reply:
Yes, we still ship them